__tests__/basic.js

```diff
@@ -13,7 +13,7 @@ test('setup', async () => {
   const { driver, pageObjects } = await setupWebDriver();
 
   await driver.wait(uiConnected(), timeouts.directLine);
-  await pageObjects.sendMessageViaSendBox('layout carousel');
+  await pageObjects.sendMessageViaSendBox('layout carousel', { waitForSend: true });
```

Comment:
`waitForSend` default to `true`, so you don't need to set it.

My general rationale around naming of option is, "put thing that is odd to the majority there (< 50%)". In this case, most of the time, you want to wait the message to send out.

I am okay to keep it for now, we will do test cleanup later.

Reply:
When I didn't set it previously the tests were failing, which is why I added them. Can I leave as is?